tasks/eth/006-MCP-L1/README.md

```diff
@@ -0,0 +1,171 @@
+# Mainnet MCP L1 Upgrade
+
+Status: DRAFT, NOT READY TO SIGN
+
+> [!IMPORTANT] !!! DO NOT SIGN using this playbook yet, as the associated governance proposal has
+> not yet passed.
+
+## Objective
+
+```markdown
+This is the playbook for executing Upgrade #6, Multi-Chain Prep (MCP) L1.
+
+The proposal was:
+- [X] Approved by Token House voting here: https://vote.optimism.io/proposals/47253113366919812831791422571513347073374828501432502648295761953879525315523
+- [ ] Not vetoed by the Citizens' house here: https://snapshot.org/#/citizenshouse.eth/proposal/0x2bc6565053b73813c6b0a001c8c07eb5656234b4d8bae12ba6541250993d1d25
+
+The proposal should be treated as the source of truth and used by the Optimism Foundation and the Collective’s
+first Security Council to verify the correctness of the onchain operations.
+
+## Signing and execution
+
+Please see the signing and execution instructions in [NESTED.md](../../../NESTED.md).
+
+## State Validations
+
+Now click on the "State" tab.
+
+- Verify that the following state overrides are made:
+   * For address `0x5a0Aae59D09fccBdDb6C6CcEB07B7279367C3d2A`:
+      * before: `0x0000000000000000000000000000000000000000000000000000000000000004`
+      * after : `0x0000000000000000000000000000000000000000000000000000000000000001`
+   * For address `0xc2819DC788505Aac350142A7A707BF9D03E3Bd03`:
+      1. before: `0x0000000000000000000000000000000000000000000000000000000000000003`
+         after : `0x0000000000000000000000000000000000000000000000000000000000000001`
+      2. before: `0x0000000000000000000000000000000000000000000000000000000000000004`
+         after : `0x0000000000000000000000000000000000000000000000000000000000000001`
+      3. before: `0x316a0aac0d94f5824f0b66f5bbe94a8c360a17699a1d3a233aafcf7146e9f11c`
+         after : `0x0000000000000000000000000000000000000000000000000000000000000001`
+      4. before: `0xe90b7bceb6e7df5418fb78d8ee546e97c83a08bbccc01a0644d599ccd2a7c2e0`
+         after : `0x000000000000000000000000ca11bde05977b3631167028862be2a173976ca11`
+
+- Verify that the following state changes are made:
+   * For address `0x229047fed2591dbec1ef1118d64f7af3db9eb290`:
+      * key `0xe314dfc40f0025322aacc0ba8ef420b62fb3b702cf01e0cdf3d829117ac2ff1a`
+         * before: `0x0000000000000000000000000000000000000005000000000000000000000000`
+         * after : `0x0000000000000000000000000000000000000006000000000000000000000000`
```

Comment:
Let's mirror the validation format used in https://github.com/ethereum-optimism/superchain-ops/blob/main/tasks/eth/004-add-superchainConfig/Validation.md, so this is easier for the signers to understand and verify. Some specifics:
- Pull addresses out into headers and explain what the address is
- Explain/justify each storage slot change, see how that was done for the linked file
- For arbitrary slot numbers like `0xe314dfc40f0025322aacc0ba8ef420b62fb3b702cf01e0cdf3d829117ac2ff1a`, explain how to calculate that slot (you'll see other validation files use `cast index` or `chisel` commands for this—either is ok)

Reply:
done! feel free to take a look